test/export/test_export.py

```diff
@@ -4327,7 +4327,7 @@ def forward(self, x, y):
         self.assertTrue(torch.allclose(ep.module()(*inp), M()(*inp)))
 
     # TODO Retracing a module with constant attrs don't work.(T193692674)
-    @testing.expectedFailureTrainingIRToRunDecomp
+    @unittest.expectedFailure
```

Comment:
The test passing was a fluke. It was earlier marked to fail only on a specific export configuration, but the test violates the assumption of export and should have failed earlier.